### DISCUSSION TITLE
Assigning Finding to a User & LDAP Configuration

### DISCUSSION THREAD

**Discussion by ravishel** [DefectDojo/django-DefectDojo#6368] [Q&A] [answered]
Hi, 
My company is planning to use DefectDojo however have 3 clarifications to make.

**1. So ideally I would like each finding to be assigned to a user (Engineer) who will resolve the issue, how do I go around doing this ?**
2. Is there LDAP Integration available.
3. I see there is integration with JIRA, as there any plans to have some sort of integration with tools like Service desk manage engine plus ?
Kind Regards

**Comment by ravishel** [comment]
Anyone please.

**Comment by mtesauro** [comment, marked as answer]
(1) If you log into the public demo server and edit an existing finding you'll find:
![image](https://user-images.githubusercontent.com/3422419/174125386-6d3fcb06-536c-4c55-abfe-546c0410934e.png)
That could be used for the purpose you're asking about.  I'd suggest using the demo server to see if it matches your needs as it could work for what you ask but isn't an exact match to your use-case.

(2) LDAP integration is possible but not supported in the UI or directly in settings.py It's been many years since I looked at LDAP support but DefectDojo is a 'normal' Django app so could be integrated with LDAP like other Django apps.  I do recall others using LDAP in the past but it's been quite a while. You might look at [this](https://django-auth-ldap.readthedocs.io/en/latest/) though the recent enhancements to the user permissions (RBAC) may make that harder than a simple Django application.  You could also employ the options listed below for (3) to resolve this issue as well.

(3) There are no current plans to integrate with other issue trackers.  However, if additional integrations are desired, you generally have 3 choices:
* Wait for the community to develop the integration (unknown timeline)
* Contribute the integration as a PR against the dev branch (you control the timeline)
* Hire someone to create the PR for you - the one company I know who does that is 10Security - see https://www.10security.com/defectdojo
Those options are the one's generally available to any open source project assuming you don't want to fork them and then own all future merges with upstream.

HTH
